c_glib/gandiva-glib/node.cpp

```diff
@@ -462,6 +459,41 @@ ggandiva_literal_node_class_init(GGandivaLiteralNodeClass *klass)
 }
 
 
+G_DEFINE_TYPE(GGandivaNullLiteralNode,
+              ggandiva_null_literal_node,
+              GGANDIVA_TYPE_LITERAL_NODE)
+
+static void
+ggandiva_null_literal_node_init(GGandivaNullLiteralNode *null_literal_node)
+{
+}
+
+static void
+ggandiva_null_literal_node_class_init(GGandivaNullLiteralNodeClass *klass)
+{
+}
+
+/**
+ * ggandiva_null_literal_node_new:
+ * @return_type: A #GArrowDataType.
+ *
+ * Returns: (nullable): A newly created #GGandivaNullLiteralNode.
+ *
+ * Since: 0.12.0
+ */
+GGandivaNullLiteralNode *
+ggandiva_null_literal_node_new(GArrowDataType *return_type)
+{
+  auto arrow_data_type = garrow_data_type_get_raw(return_type);
+  auto gandiva_node = gandiva::TreeExprBuilder::MakeNull(arrow_data_type);
+  if (gandiva_node == nullptr) {
```

Comment:
Can you use `if (gandiva_node) {return GGANDIVA_NULL_...` instead of `== nullptr`?

Reply:
Yes. I'll fixed it.